### PR TITLE
Add saga

### PR DESCRIPTION
**Pull request by ketabi**

### CODE REVIEW

File: crowd-ui/src/index.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
 import { Provider } from 'react-redux';
-import { store } from './app/store';
+import configureAppStore from './app/store';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import './index.css';
 
+const {store, } = configureAppStore();
```
Comment (ketabi) [inline crowd-ui/src/index.tsx:9]: ``` const { store, } = configureAppStore();```

File: crowd-ui/src/features/counter/saga.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { takeEvery } from 'redux-saga/effects';
+
+import {INCREMENT_ASYNC_ACTION_TYPE} from "./actions";
+
+type IncrementAsyncPayloadType = any;
+
+function* incrementAsync(action: IncrementAsyncPayloadType) {
+    console.log('saga', action);
```
Comment (masoudbigdeli) [inline crowd-ui/src/features/counter/saga.ts:8]: please add a FIXME here

**Review by kimiaShahbaghi — APPROVED** [review]

**Review by SBorouny — CHANGES_REQUESTED** [review]
change

**Review by mohanemati — APPROVED** [review]
For test

**Review by samaneteymurifar — CHANGES_REQUESTED** [review]
test

**Review by mohanemati — CHANGES_REQUESTED** [review]
test request change

**Comment by ketabi** [comment]
Please approve the changes.

**Review by mohanemati — APPROVED** [review]
approve2

**Review by kimiaShahbaghi — APPROVED** [review]

**Review by SBorouny — APPROVED** [review]

**Review by samaneteymurifar — APPROVED** [review]

**Pull request merged**